rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -476,6 +476,11 @@ namespace np
 	bool np_handler::discover_ip_address()
 	{
 		auto sockfd = socket(AF_INET, SOCK_DGRAM, 0);
+		if (!sockfd)
```

Comment:
socket returns -1 on failure.
```suggestion
		if (sockfd == -1)
```

Reply:
Lol, for some reason I thought it was a pointer.

Reply:
On windows this is actually not guaranteed to be -1 and comparison with INVALID_SOCKET is recommended

Reply:
INVALID_SOCKET is -1